{
  "file_data": [
    {
      "desc": {
        "cluster": true,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "n_out": 2
      },
      "geom": {
        "p": [
          "uinoGjctcK"
        ]
      },
      "id": "6-1",
      "title": "Area Outage"
    },
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "val": 11
        },
        "etr": "2020-12-10T23:00:00-0400",
        "start": "2020-12-10T16:23:55-0400"
      },
      "geom": {
        "a": [
          "i`shGlumoKE~G}MhBLaItMgA"
        ],
        "p": [
          "}gshGh{moK"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
